frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -51,22 +55,55 @@ export const EmbedHomepage = () => {
   }, [plan]);
 
   const onDismiss = (reason: EmbedHomepageDismissReason) => {
-    dispatch(updateSetting({ key: "embedding-homepage", value: reason }));
+    if (reason === "dismissed-run-into-issues") {
+      setFeedbackModalOpened(true);
+    } else {
+      dispatch(updateSetting({ key: "embedding-homepage", value: reason }));
+    }
+  };
+
+  const onFeedbackSubmit = ({
+    feedback,
+    email,
+  }: {
+    feedback?: string;
+    email?: string;
+  }) => {
+    dispatch(
+      updateSetting({
+        key: "embedding-homepage",
+        value: "dismiss-run-into-issues",
+      }),
+    );
+
+    setFeedbackModalOpened(false);
+    sendProductFeedback({
+      comments: feedback,
+      email: email || undefined,
```

Comment:
Why do need to to treat these 2 inputs differently? Email are treated as if we want to convert the empty string to `undefined`, but not `feedback`.

I see we already normalize both inputs the same way in `FeedbackModal.tsx`, so this could be just.

```suggestion
      comments: feedback,
      email: email,
```

Reply:
Thanks for spotting that, yeah I forgot to change it after I handled the feedback in the modal itself